docs/regex/tutorial.md

```diff
@@ -104,7 +104,7 @@ Character class | Equivalent | Pi-hole specific | Interpretation
 `[[:space:]]` | `[ \f\n\r\t\v]` | Yes | white-space characters
 `[[:xdigit:]]` | `[0-9a-fA-F]` | Yes | hexadecimal digits
 
-*) FTL matches case-insensitive by default as case does not matter in domain names
+* FTL matches case-insensitive by default as case does not matter in domain names
```

Comment:
Please revert this.
Before
![2025-04-21_10-05](https://github.com/user-attachments/assets/7f11c678-6e9b-49ca-88c2-0ebce13afc10)

After your change
![2025-04-21_10-05_1](https://github.com/user-attachments/assets/b15f8c55-a088-4858-9561-2c83478b60ab)

The asterix is the right symbol as it is the reference footnote for the table above.

Reply:
I think the markdown just needs escaping